src/utils.ts

```diff
@@ -25,29 +22,28 @@ export function mapRepositoryToQueryParams(
 
 export function mapStringToRepositoryQueryParams(
   str: string
-): RepositoryQueryParams | null {
-  try {
-    const [owner, repo] = str.split('/')
-    return { owner, repo }
-  } catch (_: unknown) {
-    return null
-  }
-}
-
-type FilterReleasesNodes = {
-  releases: Array<Release>
-  from: ReleaseVersion
-  to: ReleaseVersion
+): RepositoryQueryParams {
+  const [owner = '', repo = ''] = str.split('/')
+  return { owner, repo }
```

Comment:
I was able to simplify this one after adding tests.

Reply:
Having tests allows you to change the code and be sure you're not breaking existing functionality. It looks a lot better now!